internal/infra/entity_test.go

```diff
@@ -72,9 +71,83 @@ func Test_PopulateIntegration(t *testing.T) {
 			assert.Len(t, clusterEntity.Metrics, 1)
 
 			metrics := clusterEntity.Metrics[0].Metrics
+			require.Len(t, metrics, 3)
 
 			assert.Equal(t, clusterMetadata.Name, metrics["clusterName"])
 			assert.Equal(t, clusterMetadata.ARN, metrics["arn"])
+			assert.Equal(t, infra.EcsClusterEventType, metrics["event_type"])
+		})
+	})
+}
+
+func Test_PopulateIntegrationFargate(t *testing.T) {
```

Comment:
would make sense to just test the differences with Fargate and not the same as the test before?

Reply:
Done